lib/minheap.js

```diff
@@ -38,26 +50,48 @@ class MinHeap {
   }
 
   // This method returns true if the heap is empty
-  // Time complexity: ?
-  // Space complexity: ?
+  // Time complexity: O(1)
+  // Space complexity: O(1)
   isEmpty() {
-    throw new Error("Method not implemented yet...");
+    return this.store.length === 0;
   }
 
   // This helper method takes an index and
   //  moves it up the heap, if it is less than it's parent node.
   //  It could be **very** helpful for the add method.
-  // Time complexity: ?
-  // Space complexity: ?
+  // Time complexity: O(1) Because we're only swapping??
+  // Space complexity: O(1)?
   heapUp(index) {
```

Comment:
You're doing recursion so the time complexity is O(log n) and space complexity is the same (system stack).